core/shared/src/main/scala/com/monovore/decline/opts.scala

```diff
@@ -159,6 +160,10 @@ object Opts {
   def subcommand[A](name: String, help: String, helpFlag: Boolean = true)(opts: Opts[A]): Opts[A] = {
     Subcommand(Command(name, help, helpFlag)(opts))
   }
+
+  def envVar[A: Argument](varName: String): Opts[A] = EnvVar(varName).mapValidated(raw => {
```

Comment:
Some thoughts on the interface:
- Slight preference for just `env` over `envVar` (consistent with the other names here, which are single words; still reads fine. Thoughts? Likewise, `varName` should just be `name` for consistency.
- I think it'd be worthwhile capturing a metavar here, since `ENV_VAR=<path>` or whatever seems like useful documentation.
- We might as well capture a `help` string here too. (The only reason `argument` doesn't have one is because it doesn't have a section in the usage output to put it in... but we don't have that problem here!)